include/seastar/core/gate.hh

```diff
@@ -25,6 +25,10 @@
 #include <seastar/util/std-compat.hh>
 #include <exception>
 
+#ifdef SEASTAR_DEBUG
+#define GATE_HOLDER_DEBUG
```

Comment:
SEASTAR_ prefix to not pollute the global namespace

Reply:
fixed in v3